mosquitto/data/run.sh

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env bashio
 set +u
 
-CONFIG_PATH=/data/options.json
 SYSTEM_USER=/data/system_user.json
 
-LOGINS=$(jq --raw-output ".logins | length" $CONFIG_PATH)
-ANONYMOUS=$(jq --raw-output ".anonymous" $CONFIG_PATH)
-KEYFILE=$(jq --raw-output ".keyfile" $CONFIG_PATH)
-CERTFILE=$(jq --raw-output ".certfile" $CONFIG_PATH)
-CAFILE=$(jq --raw-output --exit-status ".cafile | select (.!=null)" $CONFIG_PATH || echo "$CERTFILE")
-REQUIRE_CERTIFICATE=$(jq --raw-output ".require_certificate" $CONFIG_PATH)
-CUSTOMIZE_ACTIVE=$(jq --raw-output ".customize.active" $CONFIG_PATH)
-LOGGING=$(bashio::info 'hassio.info.logging' '.logging')
+LOGINS=$(bashio::config ".logins | length")
```

Comment:
the first `.` is automatic extended

Reply:
Thanks!  Not too sure about the `bashio::info` one though.